src/css/style.css

```diff
@@ -1393,6 +1640,133 @@ p{
     }
     /* "form window end" */
 
+    /* "partners window start" */
+    #partnersModalWindow .close-btn-partnersPopUp{
+        left:646px;
+    }
+    #partnersModalWindow .which-contry-fields-text{
```

Comment:
3 redundant classes, why not make it a single class if does the same thing?

names aren't named according to code conventions